examples/FreshdeskWidget/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "happeofreshdeskwidget",
```

Comment:
Please rename to follow common npm package naming e.g. `happeo-freshdesk-widget`